### PR TITLE
working contact table for foodbank.js

### PR DESCRIPTION
**Pull request by andrewhojel**
working version of the contact table

* had to make some backend changes for it to work
* had to change the foodbank file to prepare for the table
* cleaned up some issues in other files

### CODE REVIEW

File: view/src/components/foodbanks.js [hunk 1]
```diff
@@ -234,10 +247,15 @@ class Foodbank extends Component {
   onTagsChange = (event, values) => {
     this.setState({
       // food bank state
```
Comment (jessimb) [inline view/src/components/foodbanks.js:249]: this comment does not seem to be about the next line, remove?

File: view/src/components/foodbanks.js [hunk 2]
```diff
@@ -856,11 +825,11 @@ class Foodbank extends Component {
...
                           </Typography>
                           <Typography variant="body2" component="p">
-                            Name: {foodbank.contactName}
+                            Name: {foodbank.contacts[0]["contactName"]}
                             <br />
-                            Phone: {foodbank.contactPhone}
+                            Phone: {foodbank.contacts[0]["contactPhone"]}
                             <br />
-                            Email: {foodbank.contactEmail}
+                            Email: {foodbank.contacts[0]["contactEmail"]}
```
Comment (jessimb) [inline view/src/components/foodbanks.js:832]: same comments as the last PR - are we sure this will be defined? do we only want the first contact?

File: view/src/components/foodbanks.js [hunk 3]
```diff
@@ -981,13 +950,13 @@ class Foodbank extends Component {
...
+                  {/* <Typography variant="body2" component="p">
+                    Name: {this.state.contacts[0]["contactName"]}
                     <br />
-                    Phone: {this.state.contactPhone}
+                    Phone: {this.state.contacts[0]["contactName"]}
                     <br />
-                    Email: {this.state.contactEmail}
-                  </Typography>
+                    Email: {this.state.contacts[0]["contactEmail"]}
+                  </Typography> */}
```
Comment (jessimb) [inline view/src/components/foodbanks.js:959]: remove - you can look back thru revision history if you really need this code again

File: view/src/components/foodbanks.js (hunk 2 above)
Comment (fatimazali) [reply view/src/components/foodbanks.js:832]: iirc when I was running this for farms, it would display a default/sample contact to show the formatting at first -- if we keep this, maybe we can be a bit more specific in the name/organization of the sample contact to make it clear that it is an example person

File: view/src/components/foodbanks.js [hunk 4]
```diff
@@ -847,22 +851,7 @@ class Foodbank extends Component {
...
-                          </Typography>
-                          <Typography variant="body2" component="p">
-                            Monday-Friday: 9am - 5pm
-                            <br />
-                            Saturday: 10am - 4pm
-                            <br />
-                            Sunday: closed
-                          </Typography>
-                        </Box> */}
+                        {/* TODO(andrewhojel): do we include hours of foodbanks? */}
```
Comment (jessimb) [inline view/src/components/foodbanks.js:854]: Nit: I'd reword this TODO to be something like "Determine if we should include hours for foodbanks"
